src/styles/style.css

```diff
@@ -69,59 +118,230 @@ body {
   font-size: 14px;
 }
 
+/* Refresh button */
 .button-recentScores {
-  font-size: 8px;
+  font-size: 20px;
   font-weight: 700;
-  border-radius: 0;
-  padding: 0 2px 0 2px;
+
   margin: 0;
+  border: none;
+  background-color: transparent;
+}
+
+.glow-on-hover {
+  outline: none;
+  color: rgb(2, 2, 2);
+  background: rgba(205, 32, 32, 0);
+  padding: 0 2px 0 2px;
+  cursor: pointer;
+  position: relative;
+  z-index: 0;
+}
+
+.glow-on-hover:before {
+  content: '';
+  background: linear-gradient(45deg, #ff0000, #020202, #ff0501, #060606);
+  position: absolute;
+  top: -2px;
+  left:-2px;
+  background-size: 400%;
+  z-index: -1;
+  filter: blur(5px);
+  width: calc(100% + 4px);
+  height: calc(100% + 4px);
+  animation: glowing 20s linear infinite;
+  opacity: 0;
+  transition: opacity .3s ease-in-out;
+}
+
+.glow-on-hover:active {
+  color: rgb(251, 251, 251)
+}
+
+.glow-on-hover:active:after {
+  background: transparent;
 }
 
+.glow-on-hover:hover:before {
+  opacity: 1;
+}
+
+.glow-on-hover:after {
+  z-index: -1;
+  content: '';
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  background: rgba(245, 242, 242, 0.015);
+  left: 0;
+  top: 0;
+}
+
+@keyframes glowing {
+  0% { background-position: 0 0; }
+  50% { background-position: 400% 0; }
+  100% { background-position: 0 0; }
+}
+
+/* List downloaded from API */
 .ul-score {
   list-style: none;
-  border: 2px solid #000;
+  border: 2px solid rgb(131, 130, 130);
+  overflow-y: scroll;
 }
 
 .ul-score li{
-  font-size: 10px;
-  padding-left: 5%;
+  font-size: 8px;
+  padding: 3px;
+  letter-spacing: 2px;
 }
 
 .ul-score li:nth-child(even){
-  background-color: rgb(221, 221, 221);
+  background-color: rgba(0, 0, 0, 0.492);
+  color: rgb(250, 250, 250);
+}
+
+.ul-score li:nth-child(odd){
+  background-color: rgba(255, 255, 255, 0.54);
+  color: #000;
 }
 
 /* table right */
+.container-addYourScore {
+  /* background-image: url(/assets/add-your-score.jpg);
+  background-size: 150% 150%;
+  background-repeat: no-repeat; */
```

Comment:
- [ ] You should make your code as clean as possible. Please remove commented code from your project!

Reply:
You're absolutely right, I got so excited working to make it look pretty that I forgot to clean up the code.

Thank you so much for the correction.